physical/cache.go

```diff
@@ -34,49 +32,43 @@ type TransactionalCache struct {
 }
 
 // Verify Cache satisfies the correct interfaces
-var _ Purgable = (*Cache)(nil)
+var _ ToggleablePurgemonster = (*Cache)(nil)
+var _ ToggleablePurgemonster = (*TransactionalCache)(nil)
 var _ Backend = (*Cache)(nil)
 var _ Transactional = (*TransactionalCache)(nil)
-var _ Purgable = (*TransactionalCache)(nil)
 
 // NewCache returns a physical cache of the given size.
 // If no size is provided, the default size is used.
-func NewCache(b Backend, size int, coreExceptions []string, logger log.Logger) *Cache {
+func NewCache(b Backend, size int, logger log.Logger) *Cache {
 	if logger.IsTrace() {
 		logger.Trace("physical/cache: creating LRU cache", "size", size)
 	}
 	if size <= 0 {
 		size = DefaultCacheSize
 	}
-	cacheExceptions := iradix.New()
-	for _, key := range coreExceptions {
-		cacheValue := true
-		if strings.HasPrefix(key, "!") {
-			key = strings.TrimPrefix(key, "!")
-			cacheValue = false
-		}
-		cacheExceptions, _, _ = cacheExceptions.Insert([]byte(key), cacheValue)
-	}
 
 	cache, _ := lru.New2Q(size)
 	c := &Cache{
-		backend:    b,
-		lru:        cache,
-		locks:      locksutil.CreateLocks(),
-		exceptions: cacheExceptions,
-		logger:     logger,
+		backend: b,
+		lru:     cache,
+		locks:   locksutil.CreateLocks(),
+		logger:  logger,
```

Comment:
`enabled` seems to be false by default (meaning no cache), is this intentional?

Reply:
Yep. Fail safe.

Reply:
Ahh, in that case perhaps adding a comment on the need to call SetEnabled before the Cache really starts caching would be helpful.

Reply:
Done